meilisearch-release-changelog/v1.7.0.md

```diff
@@ -0,0 +1,251 @@
+
+Meilisearch v1.7.0 mostly focuses on improving v1.6.0 features, indexing speed and hybrid search. GPU computing is now supported.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and improvements 🔥
+
+## Improve AI with Meilisearch (experimental feature)
+
+🗣️ AI work is still experimental and we need your help to improve it! Share your thoughts and feedback on this [GitHub discussion](https://github.com/orgs/meilisearch/discussions/677).
+
+To use it, you need to enable `vectorSearch` through [the `/experimental-features` route](https://www.meilisearch.com/docs/reference/api/experimental_features).
+
+💡 More documentation about AI search with Meilisearch [here](https://www.meilisearch.com/docs/learn/experimental/vector_search#vector-search-with-auto-embeddings).
+
+### Add new OpenAI embedding models & abbility to override their models dimensions
+
+When using `OpenAi` as `source` in your `embedders` index settings (an example [here](https://www.meilisearch.com/docs/learn/experimental/vector_search#generate-auto-embeddings-with-openai)), you can now specify two new models:
+- `text-embedding-3-small` with a default dimension of 1536.
+- `text-embedding-3-large` with a default dimension of 3072.
+
+The new models:
+- are cheaper
+- produce more relevant results in standardized tests
+- allow to set up the dimensions of the embeddings to control the trade-off between accuracy and performance (including storage)
+
+It means that it is now possible to pass the dimensions field when using the `OpenAi` source. This was previously only available for the `userProvided` source in the previous releases.
+
+There are some rules, though, we detail with these examples:
+
+```json5
+"embedders": {
+  "large": {
+    "source": "openAi",
+    "model": "text-embedding-3-large",
+    "dimensions": 512 // must be >0, must be <= 3072 for "text-embedding-3-large"
+  },
+  "small": {
+    "source": "openAi",
+    "model": "text-embedding-3-small",
+    "dimensions": 1024 // must be >0, must be <= 1536 for "text-embedding-3-small"
+  },
+  "legacy": {
+    "source": "openAi",
+    "model": "text-embedding-ada-002",
+    "dimensions": 1536 // must =1536  for "text-embedding-ada-002"
+  },
+  "omitted_dimensions": { // uses the default dimension
+    "source": "openAi",
+    "model": "text-embedding-ada-002",
+  }
+}
+```
```

Comment:
@guimachiavelli why do you say this? Did you see something in the PRD? Because when I see the examples I wrote (I copied/pasted them), you can set up dimension for `text-embedding-ada-002`

Reply:
This is for the sake of simplicity and giving users unambiguous instructions. Technically, you may set `dimensions` when using `ada-002`, but it must be exactly the model's size. Meaning, yes, you may use it, but it's not going to do anything.

We're also covering this in the docs: https://github.com/meilisearch/documentation/pull/2744#discussion_r1511475771

Reply:
From my POV, when I read "You cannot", it means you will not be able to do it at all. And it's false, you can. That's why I'm confused

Reply:
Hum, this is where I had a miss-understanding -> I thought we could put something different in `dimension`, but we can only put `1536`, I just re-read correctly [the issue](https://github.com/meilisearch/meilisearch/issues/4394)

So I agree with you!